src/main/webapp/dev/js/main/instructorHelp.js

```diff
@@ -25,7 +34,319 @@ function bindPanelCollapseLinksAndAnchor() {
     });
 }
 
+/**
+ * Constructs a hashmap of questions and utilizes it to
+ * build an index for search. Each question also has associated
+ * tags which are obtained from stemming and sub-categories allocated
+ * to different questions.
+ */
+function prepareQuestionsForSearch() {
+    let questionCount = 0;
+
+    function addQuestionsForTopic(topic) {
+        const topicContentElement = $(topic).next();
```

Comment:
currently we're using `contentHolder` as id of all topics not sure why ? If we change that to a class will the class be empty since there's no CSS rule for it atm

Reply:
You can use the class without define it. It's fine.